lib/modules/manager/azure-pipelines/extract.spec.ts

```diff
@@ -88,21 +94,69 @@ describe('modules/manager/azure-pipelines/extract', () => {
     });
   });
 
+  describe('extractAzurePipelinesTasks()', () => {
+    it('should extract azure-pipelines task information', () => {
+      expect(extractAzurePipelinesTasks('Bash@3')).toMatchSnapshot({
```

Comment:
no new snapshots

https://github.com/renovatebot/renovate/blob/main/docs/development/best-practices.md#unit-testing

Reply:
I got rid of the existing snapshots as well 🎉